client/checkout/blocks/index.js

```diff
@@ -161,12 +161,16 @@ if ( getUPEConfig( 'isTokenizedCartPrbEnabled' ) ) {
 	registerExpressPaymentMethod(
 		tokenizedCartPaymentRequestPaymentMethod( api )
 	);
-} else if ( getUPEConfig( 'isExpressCheckoutElementEnabled' ) ) {
+} else if (
+	getUPEConfig( 'isPaymentRequestEnabled' ) &&
+	getUPEConfig( 'isExpressCheckoutElementEnabled' )
+) {
 	registerExpressPaymentMethod( expressCheckoutElementApplePay( api ) );
 	registerExpressPaymentMethod( expressCheckoutElementGooglePay( api ) );
-} else {
+} else if ( getUPEConfig( 'isPaymentRequestEnabled' ) ) {
 	registerExpressPaymentMethod( paymentRequestPaymentMethod( api ) );
 }
```

Comment:
`isExpressCheckoutElementEnabled` just retrieves whether the ECE feature flag is turned on. So I created this `isPaymentRequestEnabled` variable to retrieve whether ApplePay and Google Pay are actually enabled.

In the current implementation, there are inner mechanisms that guarantee the ECE buttons won't be displayed when the setting is disabled even though they're being registered here. With this change, the payment methods won't be registered and the Blocks preview will be able to show the same as in the cart/checkout pages. @rafaelzaleski, can you double-check this change to see if there are any risks I haven't noticed?

Reply:
I don't see any risks, and thank you for making this change! Just a minor suggestion: we should [remove the PRB implementation](https://github.com/Automattic/woocommerce-payments/issues/8950) now that ECE has been around for a while. It might be better to use a new name for this flag, so we won’t need to update it again during the clean-up.